packages/webapp/pages/onboarding.tsx

```diff
@@ -427,6 +435,11 @@ export function OnboardPage(): ReactElement {
   const showOnboardingPage =
     !isAuthenticating && activeScreen === OnboardingStep.Intro && !shouldVerify;
 
+  const personalizedOnboarding = useConditionalFeature({
+    shouldEvaluate: !user && showOnboardingPage,
```

Comment:
We need to check this first. Otherwise, we will register logged-in users who accidentally landed in onboarding. This can also happen with users who did not finish their onboarding steps and got redirected.
```suggestion
    shouldEvaluate: !user && showOnboardingPage && isAuthReady,
```